lib/bolt/cli.rb

```diff
@@ -5,25 +5,20 @@
 require 'json'
 require 'bolt/node'
 require 'bolt/version'
+require 'bolt/error'
 require 'bolt/executor'
 require 'bolt/outputter'
 require 'bolt/config'
 require 'io/console'
 
 module Bolt
-  class CLIError < RuntimeError
+  class CLIError < Bolt::Error
     attr_reader :error_code
 
     def initialize(msg, error_code: 1)
-      super(msg)
+      super(msg, "bolt/cli-error")
       @error_code = error_code
```

Comment:
How does error_code get used? Should it be passed as the details argument to `super`?

Reply:
It's the exit code for bolt

Reply:
Ah, ok.